website/static/js/filepage/index.js

```diff
@@ -363,6 +363,64 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+          var editable = self.editor && self.editor.selected;
+          var viewable = self.mfrIframeParent.is(':visible');
+          if (editable || viewable){
```

Comment:
Umm, yeah I can do it that way. I just had it this way because elsewhere in the file I saw that they did this and I assumed it was to avoid extraneous if checks so I just followed that style. I can change it though.